tests/scripts/kubeadm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash +e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+tarname=image.tar
+tarfile=${WORK_DIR}/tests/${tarname}
+
+export KUBE_VERSION=${KUBE_VERSION:-"v1.6.7"}
+
+
+kubeadm_init() {
+
+    sudo kubeadm init --skip-preflight-checks
+
+    sudo cp /etc/kubernetes/admin.conf $HOME/
+    sudo chown $(id -u):$(id -g) $HOME/admin.conf
+    export KUBECONFIG=$HOME/admin.conf
+
+    kubectl taint nodes --all node-role.kubernetes.io/master-
+    kubectl apply -f https://git.io/weave-kube-1.6
+
+    kube_ready=$(kubectl get node -o jsonpath='{.items[0].status.conditions[3].status}')
+    INC=0
+    until [[ "${kube_ready}" == "True" || $INC -gt 10 ]]; do
+
+        sleep 5
+        ((++INC))
+        kube_ready=$(kubectl get node -o jsonpath='{.items[0].status.conditions[3].status}')
+    done
+
+    if [ "${kube_ready}" == "False" ]; then
+        exit 1
+    fi
+
+}
+
+kubeadm_reset() {
+    kubeadm reset
+    sudo rm /usr/local/bin/kube*
+    sudo rm kubectl
+    rm $HOME/admin.conf
+    rm -rf $HOME/.kube
+    sudo apt-get -y remove kubelet
+    sudo apt-get -y remove kubeadm
+}
+
+
+case "${1:-}" in
+  up)
```

Comment:
(for a future PR) for the local dev env, could we have this spin up a clean VM to run in? Or maybe we just expect devs to start their own VM, then run this script in it? I'd rather not run this directly on the host during development